pistrap

```diff
@@ -35,6 +36,8 @@ Enables serial console and creates user in raspberry lite image.
     Password of sudo user (default ${RASPI_PASSWORD}).
 -s|--ssh:
     Enable ssh for specified user.
+-400|--pi400:
+    Create an image for the RaspberryPi-400.
```

Comment:
I am not sure, if starting with a numeric will work, however the short parameter should be a single character.